spirv_hlsl.cpp

```diff
@@ -5603,7 +5603,7 @@ void CompilerHLSL::emit_instruction(const Instruction &instruction)
 	}
 
 	case OpIsHelperInvocationEXT:
-		if (hlsl_options.shader_model < 50 || (get_entry_point().model != ExecutionModelFragment && get_entry_point().model != ExecutionModelGLCompute))
+		if (hlsl_options.shader_model < 50 || get_entry_point().model != ExecutionModelFragment)
```

Comment:
Also remove it from the other place where this is checked, i.e. the builtin. This is the EXT opcode.

Reply:
Done.